spec/index.html

```diff
@@ -8942,15 +8942,15 @@ <h3>Basic Graph Patterns</h3>
           <a href="https://en.wikipedia.org/w/index.php?title=Multiset">multiset</a>, 
           also known as a <i>bag</i>. A multiset is an
           unordered collection of elements in which each element may appear more than once. It is
-          described by a set of elements and a cardinality function giving the number of occurrences of
-          each element from the set in the multiset.</p>
+          described by a set of elements and a function giving the multiplicity of each of these
+          elements (i.e., the number of times the element is contained in the multiset).</p>
         <p>Write μ for solution mappings.</p>
         <p>Write μ<sub>0</sub> for the mapping such that dom(μ<sub>0</sub>) is the empty set.</p>
         <p>Write Ω<sub>0</sub> for the multiset consisting of exactly the empty mapping
```

Comment:
```suggestion
        <p>Write <var>μ</var> for solution mappings.</p>
        <p>Write <var>μ<sub><var>0</var></sub></var> for the mapping such that <var>dom(<var>μ<sub><var>0</var></sub></var>)</var> is the empty set.</p>
        <p>Write <var>Ω<sub><var>0</var></sub></var> for the multiset consisting of exactly the empty mapping
```

Reply:
Some comments about your change suggestions here (although, again, I see these things as something that belongs in a PR related to #102 and #103):
* Regarding your proposed `<var>μ<sub><var>0</var></sub></var>`, this should be `<var>μ<sub>0</sub></var>` because `0` is a constant and not the name of a symbol used in the formula.
* Same thing for `<var>Ω<sub><var>0</var></sub></var>`, it should be only `<var>Ω<sub>0</sub></var>`
* Regarding your proposed `<var>dom(<var>μ<sub><var>0</var></sub></var>)</var>`, this should be `<var>dom</var>(<var>μ<sub>0</sub></var>)` -- i.e., I don't think the whole formula should be marked up as a symbol but, instead, the name of the `dom` function is a symbol in itself (and `μ<sub>0</sub>` is another symbol).

Reply:
@hartig — Thank you for your suggested fixes, all of which I have applied. I suffer from not having taken any relevant courses, and am learning some of this as we go.

Reply:
> [@TallTed] Thank you for your suggested fixes, all of which I have applied. I suffer from not having taken any relevant courses, and am learning some of this as we go.

No worries! We have several complementary types of expertise and skills in the group, and that's a good thing!

The fixes in your change suggestion look good. I will commit them.